specification/monitor/resource-manager/Microsoft.Monitor/stable/2023-04-03/monitoringAccounts_API.json

```diff
@@ -0,0 +1,704 @@
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "AzureMonitorWorkspaces"
+        ],
+        "description": "Delete a workspace",
+        "operationId": "AzureMonitorWorkspaces_Delete",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/AzureMonitorWorkspaceNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Workspace was successfully deleted"
+          },
+          "204": {
+            "description": "Workspace did not exist"
+          },
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Delete workspace": {
+            "$ref": "./examples/AzureMonitorWorkspacesDelete.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "Metrics": {
+      "description": "Information about metrics for the workspace",
+      "type": "object",
+      "properties": {
+        "prometheusQueryEndpoint": {
+          "description": "The Prometheus query endpoint for the workspace",
+          "type": "string",
+          "readOnly": true,
+          "x-ms-mutability": [
+            "read"
+          ]
+        },
+        "internalId": {
+          "description": "An internal identifier for the metrics container. Only to be used by the system",
+          "type": "string",
+          "readOnly": true,
+          "x-ms-mutability": [
+            "read"
+          ]
+        }
+      }
+    },
+    "IngestionSettings": {
+      "description": "Settings for data ingestion",
+      "type": "object",
+      "properties": {
+        "dataCollectionRuleResourceId": {
+          "description": "The Azure resource Id of the default data collection rule for this workspace.",
+          "type": "string",
+          "readOnly": true,
+          "x-ms-mutability": [
+            "read"
+          ]
+        },
+        "dataCollectionEndpointResourceId": {
+          "description": "The Azure resource Id of the default data collection endpoint for this workspace.",
+          "type": "string",
+          "readOnly": true,
+          "x-ms-mutability": [
+            "read"
+          ]
+        }
+      }
+    },
+    "PrivateEndpointConnection": {
```

Comment:
>PrivateEndpointConnection [](http://example.com/codeflow?start=5&length=25)

You can use the definitions from common-type here : [https://github.com/Azure/azure-rest-api-specs/blob/6272a882add762c6b55338da5c45a0537b812d9b/specification/common-types/resource-management/v4/privatelinks.json#L20](https://github.com/Azure/azure-rest-api-specs/blob/6272a882add762c6b55338da5c45a0537b812d9b/specification/common-types/resource-management/v4/privatelinks.json#L20)

Reply:
Thank you, I've updated the $ref value in privateEndpointConnections with "../../../../../common-types/resource-management/v4/privatelinks.json#/definitions/PrivateEndpointConnection" and deleted the nested fields.

Is this accurate?

Reply:
yes lokos good thanks . Signing off